spec/controllers/social_networking/nudges_controller_spec.rb

```diff
@@ -56,6 +56,25 @@ module SocialNetworking
             allow(controller).to receive(:send_sms) { nil }
           end
 
+          it "sends an email alert with the localized app name" do
+            post :create, recipientId: 123
+            allow(controller).to receive(:t).and_return("SunnySide")
```

Comment:
What do you think about making this slightly more specific, e.g.
`allow(controller).to receive(:t).with("application_name").and_return("SunnySide")`